cli/agent.go

```diff
@@ -40,6 +42,12 @@ func (c *AgentCommand) Execute(_ []string) error {
 		return errors.Wrap(err, "problem setting up logger")
 	}
 
+	wd, err := os.Getwd()
```

Comment:
Can we get the working directory from the distro config instead of with os.Getwd?

Reply:
no, because the agent doesn't have access to the db, so we run the command that starts the agent with the knowledge of the distro config, and then do a syscall here to get that info